### PR TITLE
Corrected django models all

### PR DESCRIPTION
**Pull request by Ebasko7**
2nd attempt on a clean branch. 

This pull request is to merge the majority of the django models with the user app that was previously merged. The pull creates 2 new apps: Class_manager_app and Course_app.

Course_app contains: Question, Answer, and Course models.

Class_manager_app contains: Curriculum, Lesson_plan, Certificate_completion, and Enrollment models.

The models themselves have defined fields, however there is more work needed to build in validation and default values.

If this pull is approved, the User model will need to be imported to the Course_app and Class_manager_app.

### CODE REVIEW

File: backend/Course_app/models.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+from django.db import models
+from django.contrib.postgres.fields import ArrayField
+
+#USER_app (User)
+class User(models.Model):
```
Comment (Star-tide) [inline backend/Course_app/models.py:5]: Lets change this to be the proper user model, since I already made that yesterday. should be user.models User

Comment (Star-tide) [inline backend/Course_app/models.py:5]: Change this to my user model

**Review by Star-tide — APPROVED** [review]
Will be handling user import at a later date

**Review by JosephL17 — APPROVED** [review]
approved

**Pull request merged**